### DISCUSSION TITLE
FileNotFoundError: /share/0813_run/init/STRU.01x01x01/02.md/sys-0016-0004-0004-0016/deepmd not found

### DISCUSSION THREAD

**Discussion by duousername1** [deepmodeling/dpgen#1617] [Q&A]
[init.zip](https://github.com/user-attachments/files/16607400/init.zip)
During the dpgen run, the corresponding folder cannot be found. However, I can navigate to the folder's path using the terminal. Could you help me figure out where the issue might be?
[0813_run.zip](https://github.com/user-attachments/files/16607397/0813_run.zip)

**Comment by njzjz** [comment]
Is there an error message showing which line throws the error?

**Reply by duousername1** [reply]
The issue has been resolved, thank you.
